aries-site/src/examples/components/avatar/AvatarDataTableIconExample.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {
+  Avatar,
+  Box,
+  DataTable,
+  Image,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+import company from '../../../data/mockData/company.json';
+
+export const AvatarDataTableIconExample = ({ bestPractice = true }) => {
+  const size = React.useContext(ResponsiveContext);
+  const [selected, setSelected] = React.useState([]);
+
+  return (
+    <>
+      <Box width="medium" height={{ max: 'large' }} overflow="auto">
+        <DataTable
+          data={company}
+          primaryKey="name"
+          columns={[
+            {
+              property: 'name',
+              header: 'Name',
+              render: datum => (
+                <>
+                  {bestPractice ? (
+                    <Box alignContent="center" gap="small" direction="row">
+                      <Avatar
+                        src={datum.src}
+                        align="undefined"
+                        round="medium"
+                        size="small"
+                      />
+                      <Text>{datum.name}</Text>
+                    </Box>
+                  ) : (
+                    <Box gap="large" alignContent="center" direction="row">
```

Comment:
```suggestion
                    <Box gap="small" align="center" direction="row">
```
the spacing shouldn't differ from the best practice, just the rendering of either Avatar or Image -- feels like we could simplify code here a bit in general